spaceship/lib/spaceship/connect_api/token.rb

```diff
@@ -97,6 +97,10 @@ def refresh!
       def expired?
         @expiration < Time.now
       end
+
+      def write_key_to_file(path)
+        File.open(path, 'w') { |f| f.write(@key) }
```

Comment:
Ahhh, this is the reason its not working! The problem is that `@key` is a `OpenSSL::PKey::EC` object. The initializer of the `Token` class needs to be rewritten to take the private key text and keep that in memory before converting it to this object 🤔  You okay with making that change?